client/rest-high-level/src/main/java/org/elasticsearch/client/security/GetUsersResponse.java

```diff
@@ -120,11 +120,11 @@ public int hashCode() {
         protected User user;
         protected boolean enabled;
 
-        public ParsedUser(String username, Collection<String> roles, Map<String, Object> metadata, Boolean enabled,
+        public ParsedUser(String username, List<String> roles, Map<String, Object> metadata, Boolean enabled,
                           @Nullable String fullName, @Nullable String email) {
             String checkedUsername = Objects.requireNonNull(username, "`username` is required, cannot be null");
-            Collection<String> checkedRoles =
-                    Set.copyOf(Objects.requireNonNull(roles, "`roles` is required, cannot be null. Pass an empty Collection instead."));
+            List<String> checkedRoles =
+                    List.copyOf(Objects.requireNonNull(roles, "`roles` is required, cannot be null. Pass an empty Collection instead."));
```

Comment:
nit: the type is no longer Collection, so the assertion message should be updated

Reply:
I pushed 7a01e7e9a30d118d8958c0c60f496fc28c6f3766.